aries-site/src/layouts/content/UpdateNotification.js

```diff
@@ -0,0 +1,58 @@
+import { Anchor, Text, Notification } from 'grommet';
+import { CircleInformation } from 'grommet-icons';
+import { ViewContext } from '../../pages/_app';
+import { useContext } from 'react';
+
+export const UpdateNotification = ({ name }) => {
+  const { wholeViewHistory } = useContext(ViewContext) || undefined;
+
+  function InlineNotification({ dateText, message }) {
```

Comment:
generally we don't nest function components inside of other ones. Can you separate InlineNotification so it's outside of UpdateNotification?

const InlineNotification ...